aws/codefresh-onprem/main.tf

```diff
@@ -0,0 +1,54 @@
+terraform {
+  required_version = ">= 0.11.2"
+
+  backend "s3" {}
+}
+
+provider "aws" {
+  assume_role {
+    role_arn = "${var.aws_assume_role_arn}"
+  }
+}
+
+variable "aws_assume_role_arn" {
+  type = "string"
+}
+
+variable "namespace" {
+  type        = "string"
+  description = "Namespace (e.g. `eg` or `cp`)"
+}
+
+variable "stage" {
+  type        = "string"
+  description = "Stage (e.g. `prod`, `dev`, `staging`)"
+}
+
+variable "region" {
+  type        = "string"
+  description = "AWS region"
+}
+
+variable "zone_name" {
+  type        = "string"
+  description = "DNS zone name"
+}
+
+data "terraform_remote_state" "backing_services" {
```

Comment:
Not sure if we need this.
@osterman what do you think?

Reply:
@goruha how else can we fetch the backing services VPC and subnets so that we can inject the CodeFresh Enterprise stuff into the same VPC/subnets?

Reply:
Yes, I like this =)